Formula/qgis3-dev.rb

```diff
@@ -244,6 +245,7 @@ def install
     args << "-DGSL_CONFIG=#{Formula["gsl"].opt_bin}/gsl-config"
     args << "-DGSL_INCLUDE_DIR=#{Formula["gsl"].opt_include}"
     args << "-DGSL_LIBRARIES='-L#{Formula["gsl"].opt_lib} -lgsl -lgslcblas'"
+    args << "-DLIBZIP_INCLUDE_DIR=#{Formula["libzip"].opt_include}"
```

Comment:
`LIBZIP_INCLUDE_DIR` still needs specifically set even though `libzip` opt prefix has been added to `CMAKE_PREFIX_PATH`?

If so, maybe we need to update the [`FindLibZip.cmake`](https://github.com/qgis/QGIS/blob/master/cmake/FindLibZip.cmake) instead?

Reply:
not sure, I applied the same fix than for the cmake script. will remove it